packages/hash/api/src/model/util.ts

```diff
@@ -100,14 +99,14 @@ export const generateSchemaBaseUri = (params: {
 }) =>
   `${params.namespaceUri}/${schemaKindSlugs[params.kind]}/${slugifySchemaTitle(
     params.title,
-  )}`;
+  )}/` as const;
```

Comment:
I believe this change was copied in from another PR that is now merged. Just flagging 
dev/graph right now:
https://github.com/hashintel/hash/blob/dev/graph/packages/hash/api/src/model/util.ts#L110

This goes for the `generateSchemaVersionedUri` change as well
https://github.com/hashintel/hash/blob/dev/graph/packages/hash/api/src/model/util.ts#L117

Reply:
The PR is up-to-date with dev/graph again so if it's still showing in the diff it's just GitHub struggling with resolving actual changes I think